src/index.js

```diff
@@ -1,9 +1,72 @@
 import './index.html';
 import './style.css';
 import { getMenuData, displayMenuItems } from './getAPI.js';
+import countMenuItems from './counter.js';
 
-const getDataAPI = 'https://www.themealdb.com/api/json/v1/1/search.php?f=f';
+const getDataAPI = 'https://www.themealdb.com/api/json/v1/1/';
+const main = document.querySelector('.homeCounter');
+const homeList = document.querySelector('.homeList');
+const allItems = document.querySelector('.menuCounter');
+const indianFood = document.querySelector('.indianFood');
+const seaFood = document.querySelector('.seaFood');
+const homeItems = document.querySelector('.menuList');
+const indianItmes = document.querySelector('.indianList');
+const seaItems = document.querySelector('.seaList');
 
-getMenuData(getDataAPI).then((meals) => {
-  displayMenuItems(meals);
+getMenuData(`${getDataAPI}search.php?f=b`).then((meals) => {
+  displayMenuItems(meals, homeList);
 });
+
+getMenuData(`${getDataAPI}search.php?f=b`).then((meals) => {
+  const mealsA = meals.meals;
+  const itemsLength = countMenuItems(mealsA);
+  allItems.innerText = `Home (${itemsLength})`;
+});
+
+getMenuData(`${getDataAPI}filter.php?c=Seafood`).then((meals) => {
+  const mealsA = meals.meals;
+  const itemsLength = countMenuItems(mealsA);
+  seaFood.innerText = `Sea Food (${itemsLength})`;
+});
+
+getMenuData(`${getDataAPI}filter.php?a=Canadian`).then((meals) => {
+  const mealsA = meals.meals;
+  const itemsLength = countMenuItems(mealsA);
+  indianFood.innerText = `Candian Food (${itemsLength})`;
+});
+
```

Comment:
You are calling these functions out of the event listener, which I see many repetitions. Following the dry and kiss rules, you should call them only once. And in this code, it's better to call them only in the add events' listener.